src/renderer/shared/ui/Countdown/Countdown.tsx

```diff
@@ -11,7 +11,7 @@ export const Countdown = ({ countdown, className }: Props) => {
   const { t } = useI18n();
 
   return (
-    <div className={cnTw('flex items-center gap-x-2 mt-2 mb-6', className)}>
+    <div className={cnTw('flex items-center gap-x-2 mt-2 mb-6 z-1', className)}>
```

Comment:
are you sure that we have `z-1`? only 0 and 10 there are in documentation. for custom value we should use `z-[1]`